lib/active_fedora/persistence.rb

```diff
@@ -146,7 +146,9 @@ def create_record(options = {})
       serialize_attached_files
       @ldp_source = @ldp_source.create
       assign_uri_to_attached_files
+      assign_uri_to_child_resources
```

Comment:
No. Child resources are like /obj1 is an RDFSource and /obj1/obj2 is also a RDFSource. "contained_rdf_source" may be more appropriate.